SPECS/atlantic-module.spec

```diff
@@ -0,0 +1,55 @@
+%define module_dir extra
```

Comment:
As discussed [in the forum](https://xcp-ng.org/forum/post/65290), this should be `updates`.